File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.java
import java.util.*;
import java.util.stream.*;

class Solution {
    public int[] solution(int[] emergency) {
        List<Integer> sortedList = Arrays.stream(emergency)
            .boxed()
            .sorted((a, b) -> b - a)
            .collect(Collectors.toList());
            
        int[] answer = new int[emergency.length];
        
        for(int i = 0; i < emergency.length; i++){
            answer[i] = sortedList.indexOf(emergency[i]) + 1;
        }
        
        return answer;
        
    }
}

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.py
def solution(emergency):
    answer = []
    
    emergency_sorted = sorted(emergency, reverse = True)
    
    for a in range(len(emergency)):
        for i in range(len(emergency_sorted)):
            if emergency[a] == emergency_sorted[i]:
                answer.append(i+1)
        
    return answer